saq/queue/postgres.py

```diff
@@ -142,6 +142,7 @@ def __init__(
         self._listener = ListenMultiplexer(self.pool, self._channel)
         self._dequeue_lock = asyncio.Lock()
         self._listen_lock = asyncio.Lock()
+        self._connected = False
```

Comment:
i thought we had something like this already

Reply:
It was removed when we introduced support for custom pools